config/locales/en.yml

```diff
@@ -391,8 +391,9 @@ en:
         SP: 'Before you upload your service provider encryption certificate:'
       you_must: "you must:"
       create_signing_key_and_certificate: "you must create a new signing key and certificate."
-      create_key_and_certificate: Create a new encryption key and certificate
+      create_key_and_certificate: "Create a new %{type} key and certificate"
```

Comment:
Shouldnt need the speech marks. Otherwise looks good to me

Reply:
aaah, good to know.. I've lived under the impression that for string interpolation it require double quotes...